src/advanced_inv.h

```diff
@@ -33,9 +33,17 @@ class advanced_inventory
         advanced_inventory();
         ~advanced_inventory();
 
+        /**
```

Comment:
Why move this doc comment? It probably refers to `enum side` where it originally was.